### PR TITLE
Added Linux Build Instructions to Documentation

### PR DESCRIPTION
**Pull request by ramesht007**
## Description
This PR introduces the Linux operating system installlation steps for PySyft library and it address the issue #4519

### CODE REVIEW

File: docs/installing.rst [hunk 1]
```diff
@@ -208,7 +208,185 @@ Congratulations! You have just installed PySyft on Mac OSX!
...
+    Python 3.8.1
+
+If the version printed underneath your command is less than 3.6, then you MUST use anaconda
+in the next step (which is recommended anyway). Technically, you could try to upgrade your
+version of python but fair warning... e're be dragons.
+
+Step 3 - Setup Environment
+--------------------------
+
+I'm sure you're tempted to skip this step. Word from the wise...
```
Comment (gmuraru) [inline docs/installing.rst:255]: I really like this :+1:

File: docs/installing.rst [hunk 2]
```diff
@@ -208,7 +208,185 @@ Congratulations! You have just installed PySyft on Mac OSX!
...
+    
+Step 4 - Install Dependencies
+-----------------------------
+
+Assuming you're still in the base directory of PySyft (as you came to be in step 1), you can
+now install the dependencies that PySyft relies on using the following command:
+
+.. code:: console
+
+    pip install -r requirements.txt
```
Comment (gmuraru) [inline docs/installing.rst:329]: Q: Shouldn't we use ```setup install -e .```?

Comment (ramesht007) [reply docs/installing.rst:329]: I used requirements.txt to install dependencies so gave that as tutorial.... if `setup install -e .` works and is better I can change that. What do you suggest?

**Review by iamtrask — APPROVED** [review]

**Comment by iamtrask** [comment]
Excellent! Thank you so much for helping buffer our the documentation! This is super important (and timely) work!

**Pull request merged**